layouts/page/cuvs.html

```diff
@@ -0,0 +1,154 @@
+</div>
+
+<section class="row section bg-rapids-gray padding-bottom-slant">
+  <div class="col">
+    <div class="container">
+      
+      <h1 class="mt-10 mb-4"> Use cases accelerated by Vector Search </h1>
+      <p class="subheading  mb-5">Featured use cases across a broad domain of applications. </p>
+
+      <div class="row">
+        <div class="col-md-3 py-3">
+          <h2><i class="fa-light fa-microchip-ai"></i> LLMs </h2>
+          <p>Vector search accelerates the querying process in large language models, leading to faster response times. By quickly finding the nearest vectors in high-dimensional spaces, it enables efficient handling of complex language patterns, enhancing these models' overall performance and responsiveness. </p>
+        </div>
+        <div class="col-md-3 py-3">
+          <h2><i class="fa-light fa-ballot-check"></i> Recommenders </h2>
+          <p>In recommender systems, vector search plays a critical role in item matching. Accelerated vector search reduces latency and improves the quality of recommendations by efficiently searching through millions of potential matches, offering users a personalized and fast experience. </p>
+        </div>
+        <div class="col-md-3 py-3">
+          <h2><i class="fa-light fa-users-viewfinder"></i> Computer Vision </h2>
+          <p>For computer vision tasks like image recognition or object detection, vector search aids in quickly finding similar vectors, thus accelerating the classification process. With GPU-accelerated vector search, the process of matching features and identifying patterns is significantly optimized, enhancing computer vision systems' overall speed and accuracy. </p>
+        </div>
+        <div class="col-md-3 py-3">
+          <h2><i class="fa-light fa-cloud-binary"></i> Data Mining </h2>
+          <p>Vector search forms the backbone for many important data mining algorithms, such as finding clusters, reducing dimensionality, and visualizing data points. GPU-accelerated vector search can take large machine learning and natural language processing workflows from hours to near real-time speeds, enabling important use-cases like hybrid search and anomaly detection. </p>
+        </div>
+      </div>
+
+
+      <h1 class="mt-10 mb-4"> Learn More</h1>
+      <p class="subheading  mb-5">Find where to get more resources and connect with developers.</p>
+
+      <div class="row">
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-light fa-flask-gear"></i> Open Beta</h2>
+          <p>cuVS is in open beta and under active development. It's ready for wide use, but has a few known limitations and edge cases. You can learn more through the <a href="https://docs.rapids.ai/api/raft/stable/" target="_blank"> documentation</a>, which also includes instructions for running <a href="https://docs.rapids.ai/api/raft/stable/raft_ann_benchmarks/" target="_blank"> cuVS ANN benchmarks</a> on your system.</p>
```

Comment:
Is this link to the RAFT docs supposed to point to https://docs.rapids.ai/api/cuvs/nightly/?

Reply:
I believe so, no stable docs yet.